### ISSUE TITLE
alias communicate missing

### ISSUE THREAD

**Issue by KriSyt** [ArmynC/ArminC-AutoExec#4]
The communicate alias that is default bound to "t" seems to be missing since
86f920a

```
[Console] Unknown command '+communicate'!
[Console] Unknown command '-communicate'!
```

readding to script.cfg solved it
```
alias +communicate "+voicerecord"
alias -communicate "-voicerecord"
```

**Comment by armync** [comment]
Nice finding :) This one (mistake) was due to Valve's devs removing the `voice_enable` cvar; therefore, I had to remove the script, overlooking the fact that the voice binding was also linked to the script, rendering it useless. Thanks!

**Comment by armync** [comment]
It should work now. Fixed in 3ee1150.
